bsmetadata/input_pipeline.py

```diff
@@ -1,38 +1,21 @@
 from dataclasses import dataclass, field
-from typing import List, Optional
+from typing import Optional
+
+from bsmetadata.metadata_utils import MetadataConfig
 
 
 @dataclass
 class DataConfig:
+    metadata_config: MetadataConfig = field(
+        default_factory=lambda: MetadataConfig(), metadata={"help": "The configuration related to the metadata."}
+    )
     experiment: str = field(default="sample", metadata={"help": "The name of the experiment."})
     per_device_eval_batch_size: int = field(
```

Comment:
Just a side note. On Colab I noticed that I can use `8` for eval.

Reply:
That a good thing to know!